### PR TITLE
Solved the task

### PR DESCRIPTION
**Pull request by Likh-Alex**
- Refactored DAO and Service using `GenericDao` and `GenericService`.
- Configured `web.xml`
- Added dependencies `servlet-api` and `maven-war-plugin` into `pom.xml`
- Created `IndexController`
- Created `index.jsp`

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/index.jsp [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+  <head>
+    <title>Title</title>
+  </head>
+  <body>
+  <h1>Hello World</h1>
+  </body>
+</html>
```
Comment (Alex-Korzun) [inline src/main/webapp/WEB-INF/views/index.jsp:9]: In checklist it's mentioned about one empty line in the end of files

File: src/main/webapp/web.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+         version="3.1">
+    <servlet>
+        <servlet-name>index</servlet-name>
+        <servlet-class>taxi.controllers.IndexController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/index</url-pattern>
+    </servlet-mapping>
+</web-app>
```
Comment (Alex-Korzun) [inline src/main/webapp/web.xml:14]: Same.
Don't know if it's necessary here.

**Review by Alex-Korzun — APPROVED** [review]

File: src/main/webapp/WEB-INF/views/index.jsp (hunk 1 above)
Comment (Likh-Alex) [reply src/main/webapp/WEB-INF/views/index.jsp:9]: I am absolutely sure that I added an empty line in the end of every file. It just does not show an additional line here.

Comment (Likh-Alex) [reply src/main/webapp/WEB-INF/views/index.jsp:9]: Screenshot 2021-01-19 at 20 47 38

Comment (Alex-Korzun) [reply src/main/webapp/WEB-INF/views/index.jsp:9]: sorry, then)

File: src/main/webapp/web.xml (hunk 2 above)
Comment (Likh-Alex) [reply src/main/webapp/web.xml:14]: Just for the sake of interest)
Screenshot 2021-01-19 at 20 48 17

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**